lms/envs/common.py

```diff
@@ -2766,3 +2766,9 @@
     'options': {},
 }
 PROCTORING_SETTINGS = {}
+
+STUDENT_RESPONSES_REPORT_SUPPORTED_TYPES = [
```

Comment:
While a `list` works, a `set` would be more appropriate, as no duplicates should exist.